ldk/go/cursorClient.go

```diff
@@ -36,25 +36,23 @@ func (c *CursorClient) ListenPosition(ctx context.Context, handler ListenPositio
 		return err
 	}
 
-	go func() {
```

Comment:
Was this just broken before?

Reply:
I think so.
The problem with these streaming calls is that, if the function returns, then gRPC closes the stream. So we need the for loop that polls for data to be blocking.

Reply:
One thing that we need to look into when we have time is if this behavior can be configured differently. There are lots of options for these gRPC connections and I think it would be better if it didn't automatically close the stream.
https://godoc.org/google.golang.org/grpc#DialOption